frontend/app/src/components/history/filtering/NoFilterAvailable.vue

```diff
@@ -107,7 +107,8 @@ export default defineComponent({
     },
     'apply:filter': (filter: string) => {
       return filter.length > 0;
-    }
+    },
+    suggest: (suggestion: Suggestion) => suggestion
```

Comment:
This should not be needed in `bugfixes` and I will probably create a conflict with `develop`.

If you got any warnings errors, you probably need to downgrade the dependencies when switching to `bugfixes`